client/go.mod

```diff
@@ -4,4 +4,23 @@ go 1.21.0
 
 require github.com/mackerelio/go-osstat v0.2.4
```

Comment:
ideally server and client should have same `go.mod` to ensure they are using same version of go packages for compatibility.